### PR TITLE
Automated cherry pick of #21440

### PR DESCRIPTION
**Pull request by bprashanth**
Cherry pick of #21440 on release-1.1.

### CODE REVIEW

**Comment by roberthbailey** [comment]
LGTM (but e2e tests need to be re-run).

File: cluster/addons/cluster-loadbalancing/glbc/glbc-controller.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: l7-lb-controller
+  name: l7-lb-controller-v0.5.2
```
Comment (dchen1107) [inline cluster/addons/cluster-loadbalancing/glbc/glbc-controller.yaml]: No tag for image here? or the name with version information will translate to image+tag?

Comment (bprashanth) [inline cluster/addons/cluster-loadbalancing/glbc/glbc-controller.yaml]: in the name? The image tag is 0.5.2, this is the pattern followed by other addons as well https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/skydns-rc.yaml.in#L4. Or did i misunderstand? 

We've alreayd cherry picked the tag bump in: https://github.com/kubernetes/kubernetes/pull/20535

**Comment by dchen1107** [comment]
I manually merged #21441 to HEAD since all checks are green for that one. But do we know why this one fail with 1.1 release branch. Also @bprashanth looks like you made some more changes after @roberthbailey's lgtm. @roberthbailey can you review it one more time based on whatever new changes here.

**Comment by roberthbailey** [comment]
@bprashanth - it looks like this doesn't apply cleanly:

```
_output/local/go/src/k8s.io/kubernetes/test/e2e/ingress.go:278: cannot use listOpts (type "k8s.io/kubernetes/pkg/api".ListOptions) as type labels.Selector in argument to cont.c.ReplicationControllers("k8s.io/kubernetes/pk:
	"k8s.io/kubernetes/pkg/api".ListOptions does not implement labels.Selector (missing Add method)
_output/local/go/src/k8s.io/kubernetes/test/e2e/util.go:2341: cannot use listOpts (type "k8s.io/kubernetes/pkg/api".ListOptions) as type labels.Selector in argument to client.ReplicationControllers(ns).List:
	"k8s.io/kubernetes/pkg/api".ListOptions does not implement labels.Selector (missing Add method)
```

**Comment by bprashanth** [comment]
I had to change listOpts for plain label selectors, because it's a new type. PTAL.

**Comment by roberthbailey** [comment]
LGTM (assuming the e2e tests pass).

**Comment by dchen1107** [comment]
e2e passed. Manually merge this one now.

**Pull request merged**